gst/nnstreamer/tensor_filter/tensor_filter_custom.c

```diff
@@ -38,6 +38,7 @@ static void init_filter_custom (void) __attribute__ ((constructor));
 static void fini_filter_custom (void) __attribute__ ((destructor));
 
 static GstTensorFilterFramework NNS_support_custom;
```

Comment:
let's remove this cuz its pointer is added.

Reply:
Updated. Removed the added pointer, and kept this instead.

Reply:
In fact, I think re-declaration of global variables is not good approach, and its compilation is prohibited in C++. But, anyway, this code works in C.

Reply:
Yes, you are right, C++ gives compilation error. I removed the re-declaration, and used pointer style.